### PR TITLE
mep12 on manual_axis.py

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

File: examples/pylab_examples/manual_axis.py [hunk 1]
```diff
@@ -5,11 +5,11 @@
 
 This example should be considered deprecated and is left just for demo
 purposes for folks wanting to make a pseudo-axis
-
 """
 
 import numpy as np
-from pylab import figure, show
+# from pylab import figure, show
```
Comment (dopplershift) [inline examples/pylab_examples/manual_axis.py]: Why comment out the line and not just remove?

Comment (ericmjl) [inline examples/pylab_examples/manual_axis.py]: Whoops! Going to remove it.

**Comment by dopplershift** [comment]
Given the comment at the top of the file, I wonder if we should just do away with this one....

**Comment by ericmjl** [comment]
Yeah, I was thinking so too... but I hesitated just to get others' input.

**Comment by WeatherGod** [comment]
It belongs somewhere... but not here. It is way too advanced for a simple pylab/pyplot example.

**Comment by ericmjl** [comment]
I will post this as an issue.

**Comment by jenshnielsen** [comment]
Since there now is an issue I will merge this

**Pull request merged**